fl4health/utils/load_data.py

```diff
@@ -79,8 +78,8 @@ def load_cifar10_data(
             transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5)),
         ]
     )
-    training_set = CIFAR10(str(data_dir), train=True, download=True, transform=transform)
```

Comment:
The `CIFAR10` that comes with `torchvision` actually doesn't work with our samplers, hence the changes here.